src/GovUk.Education.ExploreEducationStatistics.Data.Services/ViewModels/Meta/SubjectMetaViewModel.cs

```diff
@@ -1,11 +1,12 @@
 using System.Collections.Generic;
 using GovUk.Education.ExploreEducationStatistics.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Services.ViewModels.Meta.TableBuilder;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Services.ViewModels.Meta
 {
     public class SubjectMetaViewModel
     {
-        public Dictionary<string, LabelValue> Filters { get; set; }
+        public Dictionary<string, TableBuilderFilterMetaViewModel> Filters { get; set; }
```

Comment:
I'm a bit unsure about the name of the TableBuilderFilterMetaViewModel now, because we had separate naming for Table builder related classes where they were not common with the Data classes used for Tables/Charts.

Given that this one is now used by both meta view models, we could rename

Meta.TableBuilder.TableBuilderFilterMetaViewModel  -> Meta.FilterMetaViewModel
Meta.TableBuidler.TableBuilderFilterItemsMetaViewModel -> Meta.FilterItemsMetaViewModel